### PR TITLE
Mark outlook as broken

### PR DESCRIPTION
**Pull request by eloyesp**
It seems that they are rolling out per-app-passwords so they are failing to login.

### CODE REVIEW

**Review by iequidoo — APPROVED** [review]

**Review by r10s — COMMENTED** [review]
thanks a lot for taking care!

to be generated as code for the app, this  needs a before_login_hint, see https://github.com/deltachat/provider-db?tab=readme-ov-file#format and other files with state BROKEN

File: _providers/outlook.com.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 name: Outlook.com
-status: OK
+status: BROKEN
```
Comment (iequidoo) [inline _providers/outlook.com.md:3]:
```suggestion
status: BROKEN
before_login_hint: |
    Unfortunately, Outlook does not allow using passwords anymore,
    per-app-passwords are currently not working.
```

**Review by r10s — APPROVED** [review]
cannot say much about outlook, but this PR seems okay now :)

File: _providers/outlook.com.md [hunk 2]
```diff
@@ -17,6 +20,9 @@ server:
     socket: STARTTLS
     hostname: smtp.office365.com
     port: 587
-last_checked: 2022-01
+last_checked: 2024-11
 website: https://outlook.live.com/mail
 ---
+
+Unfortunately, Outlook does not allow using passwords anymore,
+per-app-passwords are currently not working.
```
Comment (r10s) [inline _providers/outlook.com.md:28]:
```suggestion
Unfortunately, Outlook does not allow using passwords anymore,
per-app-passwords are reported to not being working in November 2024.
Please let us know if you succeed using Outlook, so we can adapt these hints.
```

**Comment by iequidoo** [comment]
> cannot say much about outlook, but this PR seems okay now :)

I checked in the end of Sep, it already didn't work with usual account password. As for per-app passwords, i didn't ckeck.

**Comment by missytake** [comment]
I'd really like to check per-account passwords before merging this.

**Comment by iequidoo** [comment]
Checked it again, per-account passwords still don't work:
```
WARN deltachat_repl: src/imap.rs:419: IMAP failed to login: no response: code: None, info: Some("LOGIN failed.").
```
It stopped working in the end of Sep as they promised. Can't say anything about per-app passwords.

**Comment by missytake** [comment]
> EDIT: Tried "app passwords":
> 
> ```
> 2024-11-18T20:43:50.671287Z  WARN deltachat_repl: src/smtp.rs:146: SMTP failed to connect and authenticate: SMTP failed to login: permanent: 5.7.139 Authentication unsuccessful, basic authentication is disabled. [SCZPR80CA0013.lamprd80.prod.outlook.com 2024-11-18T20:43:50.547Z 08DD07BC036F6B0D].
> ```

Thanks a lot!

**Pull request merged**